src/pages/TeamSettings/FlowConcurrency.vue

```diff
@@ -119,7 +119,7 @@ export default {
       return this.isTenantAdmin
     },
     isTenantAdmin() {
-      return this.tenant.role === 'TENANT_ADMIN'
+      return this.hasPermission('create', 'role')
```

Comment:
I think this check should be `create:concurrency-limit`, `update:concurrency-limit`, and `delete:concurrency-limit`